packages/help-center/src/data/use-support-availability.ts

```diff
@@ -17,7 +17,7 @@ export function useSupportAvailability< SUPPORT_TYPE extends 'CHAT' | 'OTHER' >(
 	enabled = true
 ) {
 	return useQuery< ResponseType< SUPPORT_TYPE >, typeof Error >( {
-		queryKey: [ supportType === 'OTHER' ? 'otherSupportAvailability' : 'chatSupportAvailability' ],
+		queryKey: [ 'support-availability', supportType ],
```

Comment:
This one's a drive-by which simplifies the query key and fixes an eslint violation of `@tanstack/query/exhaustive-deps`.